src/sidebar/templates/annotation.html

```diff
@@ -189,21 +185,19 @@
         </annotation-share-dialog>
       </span>
       <span ng-if="vm.canFlag()">
-        <button class="btn btn-clean annotation-action-btn"
-          ng-if="!vm.isFlagged()"
-          ng-click="vm.flag()"
-          ng-disabled="vm.isDeleted()"
-          aria-label="Flag"
-          h-tooltip>
-          <i class="h-icon-annotation-flag btn-icon"></i>
-        </button>
-        <button class="btn btn-clean annotation-action-btn"
-          ng-if="vm.isFlagged()"
-          ng-disabled="vm.isDeleted()"
-          aria-label="Annotation has been flagged"
-          h-tooltip>
-          <i class="h-icon-annotation-flag annotation--flagged btn-icon"></i>
-        </button>
+        <annotation-action-button
+         icon="'h-icon-annotation-flag'"
+         is-disabled="vm.isDeleted()"
+         label="'Flag'"
+         ng-if="!vm.isFlagged()"
+         on-click="vm.flag()"
+        ></annotation-action-button>
+        <annotation-action-button
+         icon="'h-icon-annotation-flag annotation--flagged'"
+         is-disabled="vm.isDeleted()"
+         label="'Annotation has been flagged'"
+         ng-if="vm.isFlagged()"
+        ></annotation-action-button>
```

Comment:
I guess this might also work for other buttons that have logic attached to them. For example the edit and delete buttons only show if the user is authorized to edit or delete, perhaps separate `<annotation-edit-button>` and `<annotation-delete-button>` components could extract that show / don't show logic and its tests out of the annotation component